ZenPacks/zenoss/ZenPackLib/tests/test_impact_triggers.py

```diff
@@ -77,23 +67,30 @@ class ImpactTestCase(object):
     base: [SubComponent]
     monitoring_templates: [AuxComponent]
     impacted_by: [basicDevice, subComponents]
+device_classes:
+  /Server/Basic:
+    remove: true
+    zProperties:
+      zPythonClass: ZenPacks.zenoss.MockZenPack.BasicDevice
+
 """
 
 
-class TestImpactTriggers(ImpactTestCase):
+class TestImpactTriggers(ImpactTestCase, ZPLTestMockZenPack):
     """Test Impact Triggers"""
 
+    datasources = []
+    thresholds = []
+    yaml_doc = YAML_DOC
+    zenpack_name = 'ZenPacks.zenoss.MockZenPack'
+
     def afterSetUp(self):
         super(TestImpactTriggers, self).afterSetUp()
-        self.z = ZPLTestHarness(YAML_DOC)
-        self.z.connect()
+        super(ZPLTestMockZenPack, self).afterSetUp()
```

Comment:
This isn't appropriate usage of `super`. The first argument to super should also be the current class, not one of its parent classes. To invoke a method in a specific class you should use `ZPLTestMockZenPack.afterSetUp(self)`. Though in this case you should probably get rid of the useless `ImpactTestCase` parent class and use super the normal way since `ZPLTestMockZenPack` would be the only parent class.